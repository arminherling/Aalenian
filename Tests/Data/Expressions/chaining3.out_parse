AssignmentStatement: {
    Left: {
        DiscardLiteral: _
    }
    Right: {
        BinaryExpression: {
            Left: {
                NameExpression: object
            }
            Operation: Member Access
            Right: {
                FunctionCallExpression: {
                    NameExpression: func
                    ArgumentsNode(0): {
                    }
                }
            }
        }
    }
}
ExpressionStatement: {
    MemberAccessExpression: {
        FunctionCallExpression: {
            NameExpression: func
            ArgumentsNode(0): {
            }
        }
    }
}
