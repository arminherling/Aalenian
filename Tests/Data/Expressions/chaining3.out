AssignmentStatement: {
    Left: {
        Discard: _
    }
    Right: {
        BinaryExpression: {
            Left: {
                Name: object
            }
            Operation: Member Access
            Right: {
                FunctionCall: {
                    Name: func
                    Arguments(0): {
                    }
                }
            }
        }
    }
}
ExpressionStatement: {
    MemberAccess: {
        FunctionCall: {
            Name: func
            Arguments(0): {
            }
        }
    }
}
